x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/80_text.yml

```diff
@@ -1,6 +1,8 @@
 ---
 setup:
   - skip:
+      version: " - 7.10.99"
+      reason: "ESQL is available in 7.11+"
```

Comment:
Heya, is that a typo? I would think we need to `s/7/8/g`, applies to all `.yml` files.